internal/kratos/client.go

```diff
@@ -0,0 +1,29 @@
+package kratos
+
+import (
+	client "github.com/ory/kratos-client-go"
+)
+
+type Client struct {
+	c *client.APIClient
+}
+
+func (c *Client) IdentityApi() client.IdentityApi {
+	return c.c.IdentityApi
+}
+
+func NewClient(url string, debug bool) *Client {
+	c := new(Client)
+
+	configuration := client.NewConfiguration()
+	configuration.Debug = debug
+	configuration.Servers = []client.ServerConfiguration{
+		{
+			URL: url,
+		},
+	}
+
+	c.c = client.NewAPIClient(configuration)
```

Comment:
Just curious about something about the kratos client in golang. What is the default timeout configuration and does it provide http connection pooling?

Reply:
long story short I dunno, I would have to check the docs

Usually when a library is wrapping an http API they always allow to pass in (or override) the http client just in case you have specific requirements (see how AWS SDK V2 library does the signing of requests) and the pooling is done in there

I don't expect to have such a high load that would require any changes to the default http client anyway